virttest/utils_misc.py

```diff
@@ -574,7 +574,7 @@ def format_str_for_message(sr):
 
     :param str: string that will be formatted.
```

Comment:
Well str was replaced with sr, so I was just fixing the style-fix. Anyway I hate the `sr` and `str` is builtin so let me fix at least this function with a proper name instead...